content/en/real_user_monitoring/browser/monitoring_page_performance.md

```diff
@@ -147,6 +147,49 @@ The RUM SDK automatically monitors frameworks that rely on hash (`#`) navigation
 
 ## Add your own performance timing
 
+### Custom Vitals
+
+The `customVital` API allows you to measure the performance of your application at the component or the user flow level. 
+
+#### startDurationVital and stopDurationVital
+
+You can start a duration measurement by calling `startDurationVital` and stopping it later in your code with `stopDurationVital`:
+
+```javascript
+window.DD_RUM.startDurationVital("dropdownRendering")
+window.DD_RUM.stopDurationVital("dropdownRendering")
+```
+
+Once you call the `stopDurationVital` method, the custom vital duration is sent to Datadog and can be queried using `@vital.name:dropdownRendering`. For example, you can filter by duration with `@vital.duration:<10`.
+
+#### reference and description
+
+Additionally, you can use the reference returned by `startDurationVital` and specify a `description` string to support multiple instances of the same custom vital. For example, to track the duration of `dropdownRendering` in the `login` page:
+
+```javascript
+const reference = window.DD_RUM.startDurationVital("dropdownRendering", { description: "login" })
+window.DD_RUM.stopDurationVital(reference)
+```
+
+This allows you for example to visualize the Top List of the same component's rendering durations across different pages by grouping with `@vital.description`.
```

Comment:
"This allows you to view the same component's rendering behavior across different pages by grouping by '@vital.description'."